src/main/java/domain/Lottos.java

```diff
@@ -0,0 +1,58 @@
+package domain;
+
+import domain.dto.GetLottoDto;
+import domain.dto.GetLottosDto;
+import domain.dto.GetResultDto;
+import java.util.ArrayList;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map.Entry;
+
+public class Lottos {
+
+    private List<Lotto> lottos;
+
+    public Lottos() {
+        lottos = new ArrayList<>();
+    }
+
+    public void add(Lotto lotto) {
+        lottos.add(lotto);
+    }
```

Comment:
객체를 생성한 뒤, 일일이 로또를 하나씩 추가해주기보다는 외부에서 `List<Lotto>`를 만들어서  Lottos의 생성자 파라미터로 주입해주게해서 Lottos 객체를 생성자를 통해서 자신의 역할을 바로 수행할 수 있는 상태가 되도록 만들어주세요! 

그러면 객체가 불완전한 상태에서 자신의 역할을 수행하는 일이 없어질 것 같아요. 😃

```suggestion
    public Lottos(List<Lotto> lottos) {
        this.lottos = new ArrayList<>(lottos);
    }
```